newIDE/app/src/BrowserApp.js

```diff
@@ -71,6 +71,7 @@ export const create = (authentication: Authentication) => {
               renderExportDialog={props => (
                 <ExportDialog
                   project={props.project}
+                  saveProject={props.saveProject}
```

Comment:
I would call this `onSaveProject` to show that it's a callback. In the sense that it's something to call to show your intention, as the child component, to do an action, but this is just something that you ask, and you delegate the doing to another function (which is probably called saveProject in a parent somewhere)